todo/src/actions/actions.js

```diff
@@ -0,0 +1,24 @@
+export const ADD_TODO = "ADD_TODO";
+export const TOGGLE_COMPLETED = "TOGGLE_COMPLETED";
+export const REMOVE_TODO = "REMOVE_TODO";
```

Comment:
One thing that you can do is nest all of these in an object. It isn't by any means necessary, but it does make some things in your reducer nicer, in my opinion. To each their own on that front though.

Reply:
Thank you! I hadn't even considered that!